Tests/AnsiEscapesTests/AnsiEscapesTests.swift

```diff
@@ -3,13 +3,11 @@ import AnsiEscapes
 
 class AnsiEscapesTests: XCTestCase {
 	func testCursorTo() {
-		XCTAssertEqual(AnsiEscapes.cursorTo(), "\u{1B}[H")
```

Comment:
This makes no sense. What would it do without? I realize my module supports it, but no idea why. Will do the same change there.